mtp_noms_ops/apps/security/views.py

```diff
@@ -287,13 +287,13 @@ def prisoner_nomis_info_view(request, prisoner_number):
         try:
             location = get_location(prisoner_number)
             if 'housing_location' in location:
-                levels = {
-                    level['type']: level['value'] for level in
-                    location['housing_location']['levels']
-                }
-                response_data['housing_location'] = '-'.join([
-                    levels['Wing'], levels['Landing'], levels['Cell']
-                ])
+                housing_location = location['housing_location']
+                if 'levels' in housing_location:
+                    response_data['housing_location'] = '-'.join(
+                        level['value'] for level in housing_location['levels']
+                    )
+                elif 'description' in housing_location:
```

Comment:
I don't think this condition will ever get hit, as a description is always split into levels in common.